frontends/p4/specializeGenericTypes.cpp

```diff
@@ -16,51 +16,69 @@ limitations under the License.
 
 #include "specializeGenericTypes.h"
 
+#include <absl/strings/str_replace.h>
```

Comment:
unfortunately, abseil should be included using `"`:
```suggestion
#include "absl/strings/str_replace.h"
```

Reply:
hmm, unfortunate. Is there a way to coerce e.g. clang-format to enforce this?

Reply:
Not sure. And absei's checks in `clang-tidy` does not include this as well: https://clang.llvm.org/extra/clang-tidy/checks/list.html

Reply:
And for the sake of completeness, my understanding of bazel shows that `str_replace` should be in top-level `strings` library: https://github.com/abseil/abseil-cpp/blob/master/absl/strings/BUILD.bazel#L50 (`@com_google_absl//absl/strings`)